File: Selnium script for testing/Script with Python/test_scenario_10.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

def execute_test_10():
    """
    Dummy test run for scenario 10
    """
    driver = webdriver.Chrome()
    driver.implicitly_wait(10)
    driver.get("https://dummy.example.com/page_10")

    assert driver.find_element(By.ID, "success_msg").is_displayed()
    driver.find_element(By.ID, "checkout_btn").click()
    time.sleep(2)
    driver.find_element(By.ID, "user_name").send_keys("test_data_701")
    driver.find_element(By.NAME, "password_field").send_keys("test_data_386")
    driver.find_element(By.CLASS_NAME, "nav-item").click()
    assert driver.find_element(By.ID, "success_msg").is_displayed()
    time.sleep(2)
    driver.find_element(By.ID, "user_name").send_keys("test_data_640")
    time.sleep(2)
    driver.find_element(By.LINK_TEXT, "Log Out").click()
    driver.find_element(By.ID, "user_name").send_keys("test_data_673")
    driver.find_element(By.CSS_SELECTOR, ".promo-code").send_keys("test_data_273")
    driver.find_element(By.CLASS_NAME, "nav-item").click()

    driver.quit()

if __name__ == "__main__":
    execute_test_10()

File: Selnium script for testing/Script with Java/src/test/java/testscenario/TestScenario10.java
package testscenario;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;

public class TestScenario10 {
    public static void executeTest10() {
        /*
         * Dummy test run for scenario 10
         */
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.get("https://dummy.example.com/page_10");

        try {
            driver.findElement(By.className("nav-item")).sendKeys("test_data_805");
            driver.findElement(By.cssSelector(".promo-code")).sendKeys("test_data_898");
            if (!driver.findElement(By.name("password_field")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            driver.findElement(By.id("user_name")).click();
            if (!driver.findElement(By.id("checkout_btn")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.id("checkout_btn")).sendKeys("test_data_622");
            if (!driver.findElement(By.id("checkout_btn")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            if (!driver.findElement(By.cssSelector(".promo-code")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            Thread.sleep(2000);
            driver.findElement(By.id("user_name")).click();
            driver.findElement(By.id("checkout_btn")).sendKeys("test_data_508");
            driver.findElement(By.id("user_name")).sendKeys("test_data_579");
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
    
    public static void main(String[] args) {
        executeTest10();
    }
}
